modules/button/css/lib/variables.scss

```diff
@@ -1,71 +1,91 @@
 $_wdc-btn-icons-path: $wdc-system-icons-path !default;
 
-$wdc-btn-icon-size: $wdc-spacing-s;
-$wdc-btn-icon-size-l: $wdc-spacing-m;
+$wdc-btn-font-weight: 500;
```

Comment:
Can you document which variables have been removed/renamed in the PR description? We will have to add this to the migration guide.

e.g.
"Several SCSS variables have been added, removed or renamed:
- ➕ `$wdc-btn-font-weight`
- ✖️ `$wdc-btn-icon-size-l`
...
- ✏️ `$wdc-btn-bg-icon-active` > `$wdc-btn-icon-active-bg`

etc."

Reply:
will do!